subprojects/core/src/main/java/org/gradle/process/internal/streams/ExecOutputHandleRunner.java

```diff
@@ -71,6 +71,14 @@ private void forwardContent() {
         }
     }
 
+    private int readContent(byte[] buffer) {
+        try {
+            return inputStream.read(buffer);
+        } catch (IOException e) {
+            return -1;
```

Comment:
This is normal if we closed the stream or the process has been terminated (like when using the `--fail-fast` option on `Test`. I wonder if we should still log it at debug level?

Reply:
Yes, I would log at debug level at least.

Reply:
I agree: Log at debug level to be on the safe side. Just in case someone encounters unexpected behavior because of this change.